cmd/cored/main.go

```diff
@@ -290,6 +290,19 @@ func main() {
 		opts = append(opts, enableMockHSM(db)...)
 		chainlog.Printf(ctx, "Launching as unconfigured Core.")
 		h = core.RunUnconfigured(ctx, db, sdb, *listenAddr, opts...)
+
+		go func() {
+			ticker := time.Tick(5 * time.Second)
```

Comment:
@kr oh, also, do you have an opinion on the right interval? I think 5 seconds might be a little long. (Maybe 1 second?)